CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 **Highlights**
 - Support non-enum WorkflowStates to enable, for example, Kotlin workflow definitions by extending AbstractWorkflowDefinition.
+- Added `started` timestamp to workflow instance table (requires database update)
```

Comment:
Needs a rebase after 5.6.0 release and moving these changes to future release

Reply:
Done